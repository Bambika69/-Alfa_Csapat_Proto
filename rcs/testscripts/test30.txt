createfield -t safehouse
createvirologist SA1
createvirologist SA1
createequipment axe -v V1
startturn V1
toggle E1
useequipment E1 V2
useequipment E1 V2