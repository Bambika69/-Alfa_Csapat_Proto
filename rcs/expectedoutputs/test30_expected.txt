Field created:
ID: SA1
Type: Safehouse
Neighbours: null

Virologist created:
ID: V1
Field: SA1
Backpack: B1

Virologist created:
ID: V2
Field: SA1
Backpack: B2

Virologist created:
ID: V3
Field: SA1
Backpack: B3

Equipment created:
ID: E1
Type: axe
Destination: V1
Result: Successful

V1's turn started.

Equipment toggled on Virologist:
Equipment: E1
Virologist: V1
Inf: wear

Equipment used on Virologist:
Equipment:E1
Virologist: V1
Target: V2
Result: Successful
Inf: Last usage!

Equipment used on Virologist:
Equipment:E1
Virologist: V1
Target: V3
Result: Failed
